docs/src/main/asciidoc/spring-integration-support.adoc

```diff
@@ -306,12 +299,47 @@ class Demo{
 }
 ----
 
+===== Configure EventHubsMessageConverter to customize ObjectMapper
+`EventHubsMessageConverter` is made as a configurable bean to allow users to customize ObjectMapper.
+
+[#si-eh-batch]
+===== Batch Consumer Support
+To consume messages from Event Hubs in batches is similar with the above sample, besides users should set the batch-consuming related configuration options for `EventHubsInboundChannelAdapter`.
+
+When create `EventHubsInboundChannelAdapter`, the listener mode should be set as `BATCH`. And set the checkpoint mode as either `MANUAL`
+or `BATCH` as needed.
+
+[source,java]
+----
+@Bean
+class Demo{
+    private static final String INPUT_CHANNEL = "input";
+    private static final String EVENTHUB_NAME = "eh1";
+    private static final String CONSUMER_GROUP = "$Default";
+
+    public EventHubsInboundChannelAdapter messageChannelAdapter(
+            @Qualifier(INPUT_CHANNEL) MessageChannel inputChannel,
+            EventHubsProcessorContainer processorContainer) {
+        //Supported checkpoint modes are MANUAL AND BATCH only.
+        CheckpointConfig config = new CheckpointConfig(CheckpointMode.MANUAL);
+
+        EventHubsInboundChannelAdapter adapter =
+                new EventHubsInboundChannelAdapter(processorContainer, EVENTHUB_NAME,
+                        CONSUMER_GROUP, ListenerMode.BATCH, config);
+        adapter.setOutputChannel(inputChannel);
+        return adapter;
+    }
+}
+----
+
+Then a bean of `com.azure.spring.messaging.PropertiesSupplier` should be provided with the key as the event hub name, and the value as `com.azure.spring.eventhubs.core.properties.ProcessorProperties` where the batch consuming policy can be configured.
+
 [#si-eh-headers]
 ===== Event Hubs message headers
 
 The following table illustrates how Event Hubs message properties are mapped to Spring message headers. For Azure Event Hubs, message is called as `event`.
 
-.Mapping between Event Hubs Properties and Spring Headers
+.Mapping between Record Event Hubs Properties and Spring Headers
```

Comment:
Mapping between Event Hubs Message / Event Properties and Spring Message Headers in Record Listener Mode